src/olympia/ratings/templates/ratings/impala/reviews_link.html

```diff
@@ -1,7 +1,7 @@
 {% if link_to_list %}
   {% set base = url('addons.ratings.list', addon.slug) %}
 {% else %}
-  {% set base = addon.get_url_path()|urlparams('reviews') %}
+  {% set base = addon.get_url_path() + 'reviews' %}
```

Comment:
It works... but actually we can simplify it further - it resolves to the same as `{% set base = url('addons.ratings.list', addon.slug) %}` which is used if `link_to_list` is true so you can strip out if the `{%i if..` block entirely and always set base to `url('addons.ratings.list', addon.slug)`

As a follow-up issue to clean up the code, `link_to_list` could be removed from everywhere.